packages/web-components/src/components/rux-segmented-button/rux-segmented-button.scss

```diff
@@ -26,24 +26,17 @@
 .rux-segmented-button {
     display: inline-flex;
     // Default is small for 6.0 - will update default to be medium in 7.0
-    height: 1.625rem;
-    padding: 0;
-    margin: 0;
+    padding: var(--spacing-0);
+    margin: var(--spacing-0);
     list-style: none;
     border-radius: var(--radius-base);
     background-color: var(--color-background-base-default);
-    &--large {
```

Comment:
Does giving the segmented button a size prop still work after this change? ie, `<rux-segmented-button size="large">`

Reply:
Yup! I checked those out in my figma testing on the test page. The sizing I removed was redundant because `.rux-segmented-button-label` defines sizes for default, medium, and large.